spec/requests/api/v1/user_event_request_spec.rb

```diff
@@ -1 +1,52 @@
 require 'rails_helper'
+
+describe "User Events API" do
+    before :each do 
+        UserEvent.destroy_all
+    end
+    
+    it "can get all user events" do
```

Comment:
Making sure I could retrieve all of the `user_event`s if I had to. I also did this and the `show` for a `user_event` to make sure I had set up the `routes` and `controller` correctly.